statvar_imports/worldbank_ids/Schema_entity.mcf

```diff
@@ -0,0 +1,359 @@
+Node: dcid:African_Dev_Bank
+"dcid: ""African_Dev_Bank"""

```

Comment:
if you are downloading from spreadsheet, pls remove extra quotes.

Reply:
Removed extra quotes